### PR TITLE
#1 Setup Rails template

### PR DESCRIPTION
**Pull request by malparty**
#1 Setup Rails template

## What happened 👀

Basic deployment of Nimble Rails template & simple README.md update.
This is the start point for my project and enabling me to move forward. 
 
## Insight 📝

Rails Template has been configured with 2 addons:
- Devise (authentication)
- Bootstrap (UI lib)
 
## Proof Of Work 📹
image

image

### CODE REVIEW

**Comment by malparty** [comment]
Checks not passing = it's normal as this tests config will be done in #2, #3 and #4.

**Comment by ankitkalia1195** [comment]
PR title should match the issue title. 

#1 Setup Rails template

**Comment by junan** [comment]
@malparty As this PR is ready for review, we can move the related task into the `in code review section` https://github.com/malparty/google-search-ruby/projects/1

File: Procfile.dev [hunk 1]
```diff
@@ -0,0 +1,3 @@
+web: rails s -p 3000 -b '0.0.0.0'
+worker: bundle exec sidekiq -C config/sidekiq.yml
+webpack: yarn && bin/webpack-dev-server
```
Comment (junan) [inline Procfile.dev]: Same as above comment

File: .rspec [hunk 2]
```diff
@@ -0,0 +1,3 @@
+--color
+--require spec_helper
+--format documentation
```
Comment (junan) [inline .rspec]: We can add a new line

File: Procfile [hunk 3]
```diff
@@ -0,0 +1,2 @@
+web: bin/rails server -p $PORT -e $RAILS_ENV
+worker: bundle exec sidekiq -C config/sidekiq.yml
```
Comment (junan) [inline Procfile]: Same as above comment

**Review by ankitkalia1195 — APPROVED** [review]

**Review by junan — APPROVED** [review]

**Pull request merged**